src/Service/Context/PageContext.php

```diff
@@ -173,9 +177,20 @@ private function setContextByNode(Request $request) {
    *   The route object.
    * @param \Drupal\Core\Controller\TitleResolverInterface $titleResolver
    *   The title resolver.
+   * @param Drupal\Core\Language\LanguageManagerInterface $languageManager
+   *   The language manager.
    */
-  private function setContextTitle(Request $request, Route $route, TitleResolverInterface $titleResolver) {
-    // Find title.
+  private function setContextTitle(Request $request, Route $route, TitleResolverInterface $titleResolver, LanguageManagerInterface $languageManager) {
```

Comment:
`setContextTitle` has stopped being the best naming after adding language handling. As discussed, `setPageData` or `setBaseData`.

Reply:
Updated function name to ```setBaseData```